Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Brakes/MSTS/AirSinglePipe.cs

```diff
@@ -230,6 +296,15 @@ public override float GetCylVolumeM3()
         {
             return CylVolumeM3;
         }
+        public override float GetTotalCylVolumeM3()
```

Comment:
The way BrakeSystem data is stored and retrieved is pretty inconsistent, I mostly went this route because there were some existing calls to GetCylVolumeM3 that were easier to replace with another similar function call than to change to inspect a property directly.

Also, if TotalCylVolumeM3 were added as a BrakeSystem property then the other brake cylinder parameters would need to be moved as well and that starts to add clutter. Brake systems like manual braking don't need to inherit parameters that won't be used (that said, for some reason there are vacuum brake only parameters in BrakeSystem).